api/catalog/api/controllers/elasticsearch/related.py

```diff
@@ -51,9 +52,16 @@ def related_media(uuid, index, filter_dead):
 
     result_count, _ = get_result_and_page_count(response, results, page_size)
 
+    dumpable_results = (
+        results.to_dict()
+        if isinstance(results, Hit)
+        else list(map(lambda r: r.to_dict(), results))
+    )
```

Comment:
nitpick: using a list comprehension would make it more readable.
```suggestion
    dumpable_results = (
        results.to_dict()
        if isinstance(results, Hit)
        else [r.to_dict() for r in results]
    )
```

Reply:
I.... have no idea why I reached for `map` there, lol! 100% JavaScript brain, I think. I'll update this in both places. Thanks, Krystle!